sympy/physics/mechanics/joint.py

```diff
@@ -1721,13 +1721,13 @@ class SphericalJoint(Joint):
 
     A spherical joint is defined such that the child body is free to rotate in
     any direction, without allowing a translation of the ``child_point``. As can
-    also be seen in the image the ``parent_point`` and ``child_point`` are fixed
-    on top of each other, i.e. the joint_point. This rotation is defined using
-    :func:`parent_interframe.orient(child_interframe, rot_type, amounts,
-    rot_order)<sympy.physics.vector.frame.ReferenceFrame.orient>` method. The
-    default rotation consists of three relative rotations, i.e. body fixed
-    rotations. Based on the direction cosine matrix following from these
-    rotations the angular velocity is computed based on the generalized
+    also be seen in the image, the ``parent_point`` and ``child_point`` are
+    fixed on top of each other, i.e. the ``joint_point``. This rotation is
+    defined usingthe :func:`parent_interframe.orient(child_interframe, rot_type,
```

Comment:
Typo: missing space.

Reply:
Now it did exactly not fit anymore within the 80 characters